plugins/palo_alto_cortex_xdr/help.md

```diff
@@ -229,19 +265,207 @@ Example output:
 
 ### Triggers
 
+#### Get Alerts
+
+This trigger is used to get Alerts.
```

Comment:
This description isn't correct - it should fix itself after you regenerate the plugin (based on the current plugin.spec.yaml file).

Reply:
Same as before, I regenerated the plugin and get the same value here. What should it be?